instance-scheduler/main.go

```diff
@@ -268,15 +314,93 @@ func stopStartTestInstancesInMemberAccount(client IEC2InstancesAPI, action strin
 	return count
 }
 
-func getEc2ClientForMemberAccount(cfg aws.Config, accountName string, accountId string) *ec2.Client {
+func StopStartTestRDSInstancesInMemberAccount(rdsClient IRDSInstancesAPI, action string) *RDSInstanceCount {
+	action = strings.ToLower(action)
+	rdscount := &RDSInstanceCount{RDSActedUpon: 0, RDSSkipped: 0}
+
+	switch action {
+	case "start", "stop", "test":
+		break
+	default:
+		log.Print("ERROR: Invalid Action. Must be one of 'start' 'stop' 'test'")
+		return rdscount
+	}
+
+	result, err := rdsClient.DescribeDBInstances(context.TODO(), &rds.DescribeDBInstancesInput{})
+
+	if err != nil {
+		log.Print("ERROR: Could not retrieve information about Amazon RDS instances in member account:\n", err)
+		return rdscount
+	}
+
+	RDSinstancesActedUpon := []string{}
+	RDSskippedInstances := []string{}
+
+	for _, rdsInstance := range result.DBInstances {
+		var instanceSchedulingTag string
+		for _, tag := range rdsInstance.TagList {
+			if *tag.Key == "instance-scheduling" {
```

Comment:
you're not checking for autoscaling group, so remove it from the struct

Reply:
removed from the strut